.github/workflows/checks.yml

```diff
@@ -21,14 +21,16 @@ jobs:
         with:
           python-version: 3.6.8
       - uses: ./.github/actions/setup-poetry
+        with:
+          poetry-version: "1.0.0"
```

Comment:
We were using `1.0.5` any reason to use `1.0.0` now?

Reply:
1.0.5 was copied from rasa-common. It's 1.0.0 anywhere else in this project.